WeakAurasOptions/LoadOptions.lua

```diff
@@ -271,32 +295,42 @@ function OptionsPrivate.ConstructOptions(prototype, data, startorder, triggernum
             end
           end,
           set = function(info, v)
+            local payload
+            local path = appendPath(basePath, "use_" .. realname)
+            local records = {}
             if arg.multiNoSingle then
-              trigger[realname] = trigger[realname] or {};
-              trigger[realname].multi = trigger[realname].multi or {};
               if v == true then
-                trigger["use_"..realname] = false;
+                payload = false
               else
-                trigger["use_"..realname] = nil;
+                payload = nil
               end
             else
               if v then
-                trigger["use_"..realname] = true;
+                payload = true
               else
-                local value = trigger["use_"..realname];
+                local value = trigger["use_"..realname]
                 if(value == false) then
-                  trigger["use_"..realname] = nil;
+                  payload = nil
                 else
-                  trigger["use_"..realname] = false
-                  trigger[realname] = trigger[realname] or {};
+                  payload = false
                   if(trigger[realname].single) then
```

Comment:
On a new aura, this creates a lua error, because trigger[realname] is now nil, so this should be
```
                  if(trigger[realname] and trigger[realname].single) then
````